client/components/Account/AccountPage.vue

```diff
@@ -48,52 +19,26 @@
 <script>
 import ChangeAccountNameForm from '@/components/Account/ChangeAccountNameForm.vue';
 import AddCredentialForm from '@/components/Account/AddCredentialForm.vue';
-import ChangeUsernameForm from '@/components/Account/ChangeUsernameForm.vue';
-import ChangePasswordForm from '@/components/Account/ChangePasswordForm.vue';
+import AccessList from '@/components/Account/AccessList.vue';
 import DeleteAccountForm from '@/components/Account/DeleteAccountForm.vue';
-import LogoutForm from '@/components/Account/LogoutForm.vue';
 
 export default {
   name: 'AccountPage',
   components: {
     ChangeAccountNameForm,
     AddCredentialForm,
-    ChangeUsernameForm,
-    ChangePasswordForm,
+    AccessList,
     DeleteAccountForm,
-    LogoutForm,
   },
   mounted() {
-    this.$store.commit('setHeader', {title: 'Account', enableBack: true});
-  },
-  methods: {
-    async deleteCredential(username) {
-      const res = await this.$helpers.fetch('/api/accounts/credentials', {
-        method: 'DELETE',
-        body: JSON.stringify({ username })
-      });
-      if (!res) return;
-
-      this.$store.commit('setAccount', res.account);
-      if (!res.account) {
-        this.$store.commit('setUsername', null);
-        this.$router.push({ name: 'Login' });
-      }
-      this.$store.commit('alert', {
-        message: `Successfully removed ${username} from your account!`, status: 'success'
-      });
-    }
+    this.$store.commit('setHeader', {
+      title: `Shared Account Settings for ${this.$store.getters.accountName}`,
```

Comment:
Header not updating after name change; same for username in UserPage
Nitpicky but maybe more confusing than in UserPage

![image](https://user-images.githubusercontent.com/59969293/206896679-626f609f-0468-42a3-b92d-49f9b5f05589.png)

Reply:
good catch!